core/toolbox.js

```diff
@@ -684,13 +684,30 @@ Blockly.Toolbox.Category.prototype.dispose = function() {
   this.contents_ = null;
 };
 
+/**
+ * Used to determine the css classes for the menu item for this category
+ * based on its current state.
+ * @param {boolean=} selected Indication whether the category is currently selected.
+ * @return {string} The css class names to be applied, space-separated.
+ */
```

Comment:
this part just needs the "@private" then LGTM!

Reply:
Done!